### ISSUE TITLE
Poor Histogram Support for the Prometheus Exporter

### ISSUE THREAD

**Issue by SeWieland** [open-telemetry/opentelemetry-dotnet#5782]
### Package

OpenTelemetry.Exporter.Prometheus.AspNetCore

### Package Version

| Package Name                          | Version |
| ------------------------------------- | ------- |
| OpenTelemetry.Exporter.Prometheus.AspNetCore | 1.8.0-rc.1 | 
| OpenTelemetry.Extensions.Hosting | 1.9.0 | 

### Runtime Version

net6.0

### Description

The first issue I experienced: enabling exponential histograms using the `Base2ExponentialHistogramConfiguration` resulted in no histograms at all. I could not find anything in the docs, until is read #4337 

> Prometheus exporter: Currently, the exporter ignores exponential histograms. 

It would be great to mention this more prominently somwhere in the docs.
I then tried to modify the `Boundaries` of the `ExplicitBucketHistogramConfiguration` explicitly, because I need much higher values than 10000 - still the default buckets are used.

Is this simply not working, or am I missing something?

I will now try to workaround the whole issue, by pushing metrics directly via the new OTEL endpoint of prometheus instead.
If this is the recommended way because many features simply do not work with the prom exporter, please consider leaving a note in the docs as well.

### Steps to Reproduce

```csharp
builder.AddView(instrument =>
{
    return instrument.GetType().GetGenericTypeDefinition() == typeof(Histogram<>)
    ? new ExplicitBucketHistogramConfiguration(){ Boundaries = customBuckets }
    : null;
});
```

### Expected Result

histogram recordings are available in custom buckets in prometheus

### Actual Result

histogram recordings are available in default buckets in prometheus

**Comment by cijothomas** [comment]
<img width="793" alt="image" src="https://github.com/user-attachments/assets/3569e447-e684-47ba-9e8e-b922c04f56cb">
There is this warning in the doc already about this is under-development.
(And no ETA either for making it production ready)


> histogram recordings are available in custom buckets in prometheus

This is not expected. While prometheus exporter does not support Exponential, it does support explicit one, and the customer boundaries should have worked. Would you mind sharing a repro, so it can be looked at?

**Comment by cijothomas** [comment]
> Expected Result histogram recordings are available in custom buckets in prometheus

This is not expected. While prometheus exporter does not support Exponential, it does support explicit one, and the customer boundaries should have worked. Would you mind sharing a repro, so it can be looked at?

**Comment by SeWieland** [comment]
Sorry opening the issue here without investigating it entirely. I think I just got a bit frustrated with the not working `Base2ExponentialBucketHistogramConfiguration`..
By creating the minimum working example repo for debugging, I found out that it worked as expected.
Right now, I have no clue where the issue comes from in our code and will probably request to reopen the issue if it's related at all to the exporter or other otel libraries. But I think its just a misconfiguration after all.
Thanks for the heads up anyways
